tests/sparseml/transformers/sparsification/modification/test_modifying_llama.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from transformers.models.llama.modeling_llama import LlamaAttention
+
+from sparseml.pytorch.model_load.helpers import apply_recipe_structure_to_model
+from sparseml.transformers.sparsification.modification import modify_model
+
+
+@pytest.fixture
+def llama_recipe():
+    return """test_stage:
+  quant_modifiers:
+    QuantizationModifier:
+      ignore:
+        - MatMulRightInput_QK
+        - MatMulLeftInput_QK
+        - MatMulRightInput_PV
+        - MatMulLeftInput_PV
+        - MatMulOutput_QK
+        - MatMulOutput_PV
+      scheme_overrides:
+        Embedding:
+          input_activations: null
+          weights:
+            num_bits: 8
+            symmetric: False"""
+
+
+def test_modifying_llama(llama_model, shared_helper_functions):
```

Comment:
We should get to the point where we can test making a modified model and an unmodified model, and then run through them comparing outputs. This may help catch the forward passes changing between transformers versions. This may be too intensive for current hardware, but I think should absolutely be setup with the gpu runners - thoughts?

Reply:
Good idea. This is a big step toward robust detection of problems that arise from the transformers upgrade.